CODEOWNERS

```
@@ -0,0 +1,43 @@
+# OpenBao CODEOWNERS file
+
+# Everything
+* @openbao/openbao-org-maintainers
```

@driif I think this is the only line. The rest of the lines should be removed. You can't use the main repository's CODEOWNERS for this repo, each repo will need its own policy. The many of the below directories don't exist here.

---

@cipherboy does that group contain all maintainers, or only the ones that maintain the whole of the openbao/openbao repository?  What about the jwt committers?

Really it doesn't make sense to me to have a CODEOWNERS file at all, instead of just handling it at the github configuration level.

---

Ah, sorry, I meant to provide a suggestion that it should be:

 ```suggestion
* @openbao/openbao-org-maintainers @openbao/openbao-auth-jwt-committers
```

But it only includes the whole-org-level maintainers, not the new committer promotions to the `vault`/`core` team on the main repository: https://github.com/openbao/openbao/blob/main/MAINTAINERS.md#organization-level-maintainers

So just Dan, Jan, Nathan, and myself. Plus you in the JWT committers (now that that is fixed).

See my thoughts on why there's some value in both CODEOWNERS and GitHub settings here: https://github.com/openbao/dev-wg/issues/50#issuecomment-3012938692